python/paddle/fluid/tests/unittests/distributed_passes/auto_parallel_pass_test_base.py

```diff
@@ -63,12 +63,12 @@ def apply_no_passes(self):
 
     def check_main(self, gpus=None, **kwargs):
```

Comment:
I do not know why we really need a class which has so many duplicate codes with `DistPassTestBase`.

Reply:
we want to rewrite the member functions: "_run_gpu_main":  since we want the check how the pass co-operate with other auto_parallel logic(like graph partition), so we need to call this pass from fleet_base where will trigger both auto parallel and this pass. we could not re-use the _run_gpu_main in DistPassTestBase.